cfme/fixtures/v2v_fixtures.py

```diff
@@ -416,11 +418,12 @@ def get_vm(request, appliance, source_provider, template, datastore='nfs'):
     source_datastore = [d.name for d in source_datastores_list if d.type == datastore][0]
     collection = source_provider.appliance.provider_based_collection(source_provider)
     vm_name = random_vm_name("v2v-auto")
+    template_name = _get_template(source_provider, template).name
```

Comment:
Q: Why not just use the template dict and not just the name attr? It will result in less changes to the existing code. If you'd like to leave it as is, why not have `_get_template` only return the name? Then call it `get_template_name`.

Reply:
Also I think the `template` var should be consistently named in this method. Since you use `template_type` below, how about that?